scripts/tuple_tree_generator/tuple_tree_generator/generators/cppheaders.py

```diff
@@ -202,6 +202,7 @@ def _emit_impl(self):
                         schema=self.schema,
                         root_type=self.root_type,
                         base_namespace=self.schema.base_namespace,
+                        emit_tracking=self.emit_tracking,
```

Comment:
Why is this in this commit?

Reply:
Because, apparently, nobody used track guards in `Impl.cpp` before this. See the corresponding changes to the relevant template.